lib/nxt_support/preprocessors/strip_preprocessor.rb

```diff
@@ -0,0 +1,18 @@
+module NxtSupport
+  module Preprocessors
+    class StripPreprocessor
+      attr_accessor :value
+
+      def initialize(value)
+        @value = value
+      end
+
+      def call
+        return value unless value.is_a?(String)
```

Comment:
Same here. Don't think you should silently ignore this.

Reply:
You could even have a ImplementsMethodPreprocessor super class or something that checks if the value implements the method that you want to call upon it and raises a nice error saying that you wanted to call a method on something that does not implement it. Or you simply rescue from method not implemented and raise your own error.